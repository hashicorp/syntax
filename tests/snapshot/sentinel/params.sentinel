# https://developer.hashicorp.com/sentinel/docs/language/parameters

param foo
param bar default 42

param someting default [1,2,3]
