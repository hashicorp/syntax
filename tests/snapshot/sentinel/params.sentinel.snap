># https://developer.hashicorp.com/sentinel/docs/language/parameters
#^ source.sentinel comment.line.number-sign.sentinel punctuation.definition.comment.sentinel
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.sentinel comment.line.number-sign.sentinel
>
>param foo
#^^^^^ source.sentinel keyword.control.declaration.sentinel
#     ^^^^^ source.sentinel
>param bar default 42
#^^^^^ source.sentinel keyword.control.declaration.sentinel
#     ^^^^^ source.sentinel
#          ^^^^^^^ source.sentinel keyword.other.sentinel
#                 ^ source.sentinel
#                  ^^ source.sentinel constant.numeric.number.sentinel
>
>param someting default [1,2,3]
#^^^^^ source.sentinel keyword.control.declaration.sentinel
#     ^^^^^^^^^^ source.sentinel
#               ^^^^^^^ source.sentinel keyword.other.sentinel
#                      ^^ source.sentinel
#                        ^ source.sentinel constant.numeric.number.sentinel
#                         ^ source.sentinel
#                          ^ source.sentinel constant.numeric.number.sentinel
#                           ^ source.sentinel
#                            ^ source.sentinel constant.numeric.number.sentinel
#                             ^^ source.sentinel
>